Tools/Scripts/generate-bundle

```diff
@@ -484,7 +484,9 @@ class BundleCreator(object):
     def _get_mesa_libraries(self):
         mesa_library_names = ['libglapi', 'libEGL', 'libGL', 'libGLESv2', 'libGLX', 'libdrm', 'libgbm', 'libMesaOpenCL']
         mesa_libraries = []
-        lib_dir_dri = self._get_pkg_config_var('dri', 'libdir')
+        dridriver_dir = self._get_pkg_config_var('dri', 'dridriverdir')
+        lib_dir_dri = os.path.dirname(dridriver_dir)
+
```

Comment:
This will not be correct on all the cases. For example, inside the flatpak SDK we have this:

```
# cat /usr/lib/x86_64-linux-gnu/pkgconfig/dri.pc
prefix=/usr
libdir=/usr/lib/x86_64-linux-gnu

dridriverdir=/usr/lib/x86_64-linux-gnu/GL/lib/dri

Name: dri
Description: Direct Rendering Infrastructure
Version: 23.1.3
```

So if `libdir` is defined we want to use that, only when not defined we can guess is the directory above.
Suggested change:

```
try:
    lib_dir_dri = self._get_pkg_config_var('dri', 'libdir')
except RuntimeError:
    lib_dir_dri = os.path.dirname(self._get_pkg_config_var('dri', 'dridriverdir'))
```